controllers/article.js

```diff
@@ -23,7 +23,7 @@ class Article {
         title, body, taglist
       } = req.body;
       const slug = slugInstance.returnSlug(title);
-      const authorid = 100;
+      const authorid = req.user;
```

Comment:
Good job @jnkindi, something i would like to add on line 26, if you're getting the user from the a middleware, shouldn't it be an object. Writing **req.user** to me, doesn't seem to be the **author  id**. I would say that it would rather be **req.user.id** which is more meaningful.

Reply:
@frankhn, In the middleware the user id is sent in the request as an attribute 'user' to req. It is not an object but an integer.